src/cases/components/case-hearings-list/case-hearings-list.component.html

```diff
@@ -43,20 +43,21 @@
            href="javascript:void(0)"
            (click)="cancelHearing(hearing.hearingID)">Cancel</a>&nbsp;&nbsp;&nbsp;&nbsp;
       </div>
-      <div *ngIf="hasUpdateAction && !isNonCancellable(hearing.exuiDisplayStatus) && hearing.hearingIsLinkedFlag && !isManageLinksEnabled(hearing.hearingGroupRequestId)" class="div-action">
+      <div *ngIf="hasUpdateAction && !isNonCancellable(hearing.exuiDisplayStatus) && hearing.hearingIsLinkedFlag && !isManageLinksEnabled(hearing.hearingGroupRequestId) && isAwaitingListingOrUpdateRequestedOrListed(hearing.exuiDisplayStatus)" class="div-action">
```

Comment:
Could we modify this isAwaitingListingOrUpdateRequestedOrListed to more generic like 'isInLinkableStatus'. i.e. if BE add one more status we have to amend this method to add *OrOtherStatus.